client/my-sites/site-settings/disconnect-site/index.jsx

```diff
@@ -9,28 +9,123 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
+import { CompactCard } from '@automattic/components';
 import DisconnectSurvey from './disconnect-survey';
 import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
+import Gridicon from 'components/gridicon';
 import Main from 'components/main';
 import NavigationLink from 'components/wizard/navigation-link';
 import redirectNonJetpack from 'my-sites/site-settings/redirect-non-jetpack';
 import Troubleshoot from './troubleshoot';
-import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { getSelectedSiteSlug, getSelectedSite } from 'state/ui/selectors';
+import { localizeUrl } from 'lib/i18n-utils';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const DisconnectSite = ( { reason, siteSlug, translate } ) => {
-	const confirmHref = '/settings/disconnect-site/confirm/' + siteSlug;
+const DisconnectSite = ( {
+	reason,
+	type,
+	siteSlug,
+	site: { name: siteName, URL: siteUrl },
+	translate,
+} ) => {
+	const confirmHref = `/settings/disconnect-site/confirm/${ siteSlug }?type=down`;
 
 	let backHref = '/settings/manage-connection/' + siteSlug;
 	if ( reason ) {
 		backHref = '/settings/disconnect-site/' + siteSlug;
 	}
 
+	if ( type === 'down' ) {
+		return (
+			<div>
+				<Main className="disconnect-site__down-flow">
+					<DocumentHead title={ translate( 'Site Settings' ) } />
+					<FormattedHeader
+						headerText={ translate(
+							'Your site {{strong}}%(siteName)s{{/strong}} cannot be accessed',
+							{
+								args: { siteName },
+								components: { strong: <strong /> },
+							}
+						) }
+						subHeaderText={ translate(
+							'Jetpack wasn’t able to connect to your site at {{strong}}%(siteSlug)s{{/strong}}.{{br/}}There might be a few reasons for that, how do you want to proceed?',
+							{
+								args: { siteSlug },
+								components: {
+									strong: <strong />,
+									br: <br />,
+								},
+							}
+						) }
+					/>
+					<div className="disconnect-site__actions">
+						<CompactCard href={ siteUrl } target="_blank" rel="noopener noreferrer">
```

Comment:
design: unlike the figma prototype, these are link-coloured and normal font-weight, is that ok?

Reply:
(i’ve used href cards here because all of these options have a URL, so i think we should let users do anything they can do with other links (middle-click new tab, right-click bookmark, …), but we can definitely override the appearance or do this another way)

Reply:
Thanks for this change! Having the link color there actually gives this more separation between the options and make them stand out a bit more. Let's go with this.